docs/articles/new-expensify/billing-and-subscriptions/adding-payment-card-subscription-overview.md

```diff
@@ -0,0 +1,31 @@
+[title] Subscription Management
+Under the subscriptions section of your account, you can manage your payment card details, view your current plan, add a billing card, and adjust your subscription size and renewal date.
+To view or manage your subscription in New Expensify:
+**Open the App**: Launch New Expensify on your device.
+**Go to Account Settings**: Click your profile icon in the bottom-left corner.
+**Find Workspaces**: Navigate to the Workspaces section.
+**Open Subscriptions**: Click Subscription under Workspaces to view your subscription.
+
+## Add a Payment Card
+
+Look for the option to **Add Payment Card**. Enter your payment card details securely to ensure uninterrupted service.
+[PLACEHOLDER for design image- default]
+## Subscription Overview
+
+This is where you can view your current subscription plan and see details like the number of seats, billing information, and the next renewal date.
+
+**Subscription Settings**:
+   - **Auto-renew**: See when your subscription will automatically renew (e.g., **Renews on Nov 1, 2024**).
+- **Auto-increase annual seats**: Here you can see how much you could save by automatically increasing seats to accommodate team members who exceed the current subscription size.
+
+**Note**: This will extend your annual subscription end date.
+[PLACEHOLDER for design image- your plan]
```

Comment:
Hey @zanyrenney some if this is showing this placeholder texts, should I merge it anyway?

Reply:
Actually I now have merged the help site images so let me amend this real quick before we merge

Reply:
Hmmm I think this could probably be done in a different order:

1. Upload images
2. Write the article with the images

so we don't have placeholder texts, anyway just a suggestion, I am gonna go ahead and merge then.